data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+			choice
+				`	"I will get to it right away then."`
+					accept
+				`	"I do not want to bring you Hai defectors, sorry."`
+					decline
+	
+	on stopover
+		dialog `The Hai seem happy to get on your ship, now return to <origin> with them.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		clear "Unfettered: offered bring hai"
+		log `Helped some Hai citizens to join the Unfettered.`
+		payment 100000
+		dialog `The Unfettered thank you for helping these "True" Hai find their way home, and give you <payment>. After discussing with them, the Unfettered elder says, "If you want to help us again, the hospitals on Darkcloak are in grave need of resupply." Checking back on the group, you can see they are doing fine, and already finding a place to call home, shared among the others. They may not have much, but they seem not to require anything at all, other than to be here.`
+
+
+
+mission "Unfettered: Bring Medical Supplies Notification"
+	landing
+	name `Need for medical supplies`
+	description `An Unfettered elder told you to come help the hospitals on <destination> that desperately need supplies.`
+	source "Firelode"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Bring Hai: done"
+
+
+
+mission "Unfettered: Bring Medical Supplies"
+	landing
+	name `Bring medicals to <planet>`
+	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	source "Darkcloak"
+	stopover "Snowfeather"
+	cargo "medicals" 120
+	blocked `You need 120 cargo space available to carry the medicals supplies.`
+	deadline 8
+	to offer
+		has "Unfettered: Bring Hai: done"
+	on offer
+		conversation
+			`As you land, you receive a distress message, originating from one of the hospitals nearby. Staff are asking all who can to immediately bring back medical supplies from <stopovers>.`
+			`	When landing on an Unfettered planet, even after you brought them a jump drive, there is always some lingering tension. Now you feel the tension much more than usual, probably because of the medical emergency. Unfettered are all around, launching small ships to bring the wounded to the hospitals as quickly as possible. The hospital itself looks different than the rest of the Unfettered structures, it seems more organized, proper. It is probably the best maintained structure, or at least has that appearance, because it is the most useful to them, alongside the shipyards. But one part of it looks to have been grafted onto the rest, and not by a well trained physician, to say the least... You figure they must really be needing more space for their sick.`
```

Comment:
Is there a recent battle or accident? This is far too hectic to be business as usual, they'd have collapsed long before now if this is a common standard.

Reply:
There is an epidemic proceeding, and with their limited founds and resources they fully realize how dangerous that can be if they do not act quickly. We can assume this is something that happened in the past and did not go well, probably not on this world though.

Reply:
It was supposed to have happened whilst you were away doing the other mission, I never touched on the news and they just started to notice it pretty much at the time you arrive.
Would be weird to do an event to add news for a week to then just remove it?
This isn't covid situation or whatever not very harmfull diseases there could be out there, think more of a pest that kills quickly, spreads quickly, and then ends quickly, with half the people dead if you don't do a thing.
This isn't the first one considering they often get Hai joining them, Hai who saw a lot of different aliens when the UHai stay by themselves in general.

Reply:
You get sent to the world - you can do news stuff if you like or you can just do a text only "news" mission where you hear about it on the news, or someone can just straight up tell you, but the point is _when you get sent_ to that world, you should explicitly be told why you're going there - you're bringing supplies _for an epidemic_.

Reply:
I included it in the distress message, and gave more depth to the mission, to make it fit UHai a bit more.